pygmt/src/histogram.py

```diff
@@ -10,11 +10,20 @@
     A="horizontal",
     B="frame",
     C="cmap",
+    D="annotate",
+    E="bar_width",
+    F="center",
     G="fill",
     J="projection",
+    N="normal",
+    Q="cumulative",
     R="region",
+    S="steps",
```

Comment:
Use singular instead of plural?
```suggestion
    S="step",
```